apps/web/src/pages/[document].tsx

```diff
@@ -15,7 +15,8 @@ import {
   encryptPage,
   serializePage,
 } from "../utils/page-helper";
-
+import { Edit } from "lucide-react";
+import { Button } from "../components/ui/button";
```

Comment:
Yes! I actually removed that separate package in this PR since I deemed it to be overhead for this project atm. But whenever we rebuild some of the core i definitly think it should live in a separate package